pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[tool.poetry]
+name = "csrank"
+version = "1.2.1"
+description = "Context-sensitive ranking and choice"
+authors = ["Karlson Pfannschmidt <kiudee@mail.upb.de>"]
+license = "apache-2.0"
+classifiers = [
+    "Development Status :: 4 - Beta",
```

Comment:
Is this still accurate? If we're following semver, 1.x shouldn't be considered beta right?